src/AirshipBindings.Android.Core/AirshipBindings.Android.Core.csproj

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\AirshipBindings.NETStandard\packages\Xamarin.AndroidX.Migration.1.0.6\build\monoandroid90\Xamarin.AndroidX.Migration.props" Condition="Exists('..\AirshipBindings.NETStandard\packages\Xamarin.AndroidX.Migration.1.0.6\build\monoandroid90\Xamarin.AndroidX.Migration.props')" />
```

Comment:
I _hate_ how VS references other projects in this way! There is no way Core should depend on the NETStandard folder. If you remove `..\AirshipBindings.NETStandard\` from the front of all these lines, it will be much cleaner. It looks like there are similar issues in the other projects.